fuse_examples/imaging/oai_example/downstream/classification_config.yaml

```diff
@@ -0,0 +1,40 @@
+defaults:
+  - _self_
+  - override hydra/hydra_logging: disabled
+  - override hydra/job_logging: disabled
+
+hydra:
+  output_subdir: null
+  run:
+    dir: .
+
+task : classification
```

Comment:
Add in comments on the non-trivial arguments together with the available options.